### ISSUE TITLE
Add another button to Add this task and create new one!

### ISSUE THREAD

**Issue by ctoabidmaqbool** [flyspray/flyspray#880]
Currently when user click on **Add this task** in `Add new task` page, then page is directly switched to `Listlist` page.

![image](https://github.com/flyspray/flyspray/assets/10474725/5f6d13ae-7d16-4cd4-99dd-1697d18febb4)

There should be an option to remains to the same page e.g. when trying to add multiple tasks. As user have to click on `Add new task` page button again!

**Comment by CaraesNaur** [comment]
That's what "Add Multiple Tasks" (the next tab) is for.

**Comment by ctoabidmaqbool** [comment]
@CaraesNaur Yeah! I was not looking to add multiple tasks at a same time, rather adding multiple taks one after other!

**Comment by CaraesNaur** [comment]
It would be a simple and highly effective change, especially considering how clunky the "Add Multiple Tasks" interface can be.

If users could automatically repeat adding tasks, then I could see "Add Multiple Tasks" being replaced with an "Import Tasks" workflow where a CSV file gets column-mapped and uploaded.

When I started using Flyspray, I had 80 tasks ready to go.  I wrote a script to inject them directly into the database.  Not something a typical user can be expected to do.

**Comment by peterdd** [comment]
I am not a fan of the current 'Add multiple tasks' feature implementation and the button so prominently placed in the top menu. Thinking of moving them out of that menu if an alternative exist.

It has to check if all of the tasks infos given are correct and not duplicates etc and if one task has 'errors' has to abort the whole form losing maybe inserted stuff and users mind and time. (or need some ajax preview checks added)

One simple idea is to have a button beside the 'Add this task' button named like 'Add this task and add another in this project' or something like that. But it's probably a bit ugly too...

I am open for ideas.

**Comment by CaraesNaur** [comment]
Interestingly, the translation text for this ("Add another task after this one") has been present since 2006.

Was this feature never implemented, or removed at some point?

**Comment by CaraesNaur** [comment]
@peterdd Expect a PR that implements this in a little while.

**Comment by Neustradamus** [comment]
Yes, it will be nice to have a button to create a new task after to validate the current one.
